packages/components/src/unit-control/types.ts

```diff
@@ -24,3 +38,61 @@ export type WPUnitControlUnit = {
 };
 
 export type WPUnitControlUnitList = Array< WPUnitControlUnit > | false;
+
+export type UnitControlOnChangeCallback = InputChangeCallback<
+	SyntheticEvent< HTMLSelectElement | HTMLInputElement >,
+	{ data?: WPUnitControlUnit }
+>;
+
+export type UnitSelectControlProps = {
+	/**
+	 * Whether the control can be focused via keyboard navigation.
+	 */
+	isUnitSelectTabbable: boolean;
```

Comment:
This prop shouldn't be marked as required

```suggestion
	isUnitSelectTabbable?: boolean;
```

Reply:
Ah, yes, good catch. Committed!